int64 request
---
int64 error_code